Changelog.md

```diff
@@ -15,6 +15,7 @@
 - Fixed ordering of assignments in the Assignment dropdown menu and Assignment index page (#7642)
 - Updated Section model associations with appropriate dependent options to handle cascade deletion while preventing deletion when students exist (#7681)
 - Prevent grade change in the grades table for a Marks Spreadsheet, when scrolling up or down with mouse or keys (#7680)
+- Fixed ordering of sections in the section-specific settings table (#7674)
```

Comment:
The number is incorrect: it should refer to the pull request number (7690), not the issue number.

Also please mention that this change is on the assignment settings form.